src/scenes/ProgressReports/EditForm/Steps/SubmitReportStep/ConfirmVariantDialog.tsx

```diff
@@ -64,20 +63,20 @@ export const ConfirmEmptyVariants = ({
     roles
   );
   if (!hasSelectedTeamNewsPrompt || emptyTeamNewsResponsesByRole.length > 0) {
```

Comment:
IMO these double ifs and function calls could be simplified a bit IE:

  const listItemsData = [
    {
      key: "teamNews",
      condition: !report.teamNews.items.length || getEmptyResponsesForRole(report.teamNews, roles).length,
      label: "Team News"
    },
    {
      key: "story",
      condition: !report.communityStories.items.length || getEmptyResponsesForRole(report.communityStories, roles).length,
      label: "Story"
    }
  ];

  const listItems = listItemsData.filter(item => item.condition).map(item => (
    <li key={item.key}>{item.label}</li>
  ));